Changelog.md

```diff
@@ -10,6 +10,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Control Flow Graph: Detect every access to uninitialized storage pointers.
```

Comment:
hm, shouldn't it rather be a feature?

Reply:
If we consider it a feature, then one could argue that it's breaking, that's why I put it here :-). But I can move it.

Reply:
ok :)